staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3815,6 +3828,7 @@ type PodStatus struct {
 	// +optional
 	// +patchStrategy=merge
 	// +patchMergeKey=ip
+	// +listType=set
```

Comment:
Why did we change that?

Reply:
https://github.com/kubernetes/kubernetes/pull/101566#r793154264

for remove
```
API rule violation: list_type_missing,k8s.io/api/core/v1,PodStatus,PodIPs
```
in `api/api-rules/violation_exceptions.list`

Reply:
https://github.com/kubernetes/kubernetes/pull/101566/commits/302f9274be3e8d6e43852e9a0957cf166a55eca5

Reply:
I think we can;t change existing ones but we can stop making new ones bad.